modules/helm/crds.mk

```diff
@@ -42,7 +42,7 @@ endif
 ## @category [shared] Generate/ Verify
 generate-crds: | $(NEEDS_CONTROLLER-GEN) $(NEEDS_YQ)
 	$(eval crds_gen_temp := $(bin_dir)/scratch/crds)
-	$(eval directories := $(shell ls -d */ | grep -v '_bin' | grep -v 'make'))
+  $(eval directories := $(shell ls -d */ | grep -v -e '_bin' -e 'make' $(shell git check-ignore -- * | sed 's/^/-e /')))
```

Comment:
Does this work on both linux and darwin?
What machine did you use to test this @maelvls?

Reply:
After re-reading this code, I think this works on both platforms. Linux/ darwin struggles mostly with the `-i` flag for `sed`.